src/index.js

```diff
@@ -1,6 +1,8 @@
 require('./require-angular-injections.js');
 
 require('./linagora.esn.videoconference/app/videoconference.module.js');
+require('./linagora.esn.videoconference/app/videoconference.config.js');
 require('./linagora.esn.videoconference.calendar/app/videoconference-calendar.module.js');
+require('./linagora.esn.videoconference.calendar/app/videoconference-calendar.config.js');
```

Comment:
This file should be required inside `./linagora.esn.videoconference.calendar/app/videoconference-calendar.module.js`